scripts/index.js

```diff
@@ -1,7 +1,136 @@
-function IsPositive(number) {
-  if (number > 0) { return true; }
+// ============================================================================================
+// Annimating the menu button
+let i = 0;
+
+const btnMenu = document.getElementById('menu_icon');
+const btnCloseMenu = document.getElementById('close_menu_icon');
+
+const mobileMenu = document.getElementById('mobile_menu');
+
+const btnPopUp = document.getElementsByClassName('see');
+const popUpWindows = document.getElementById('pop_up');
+
+btnMenu.addEventListener('click', () => {
+  mobileMenu.className = 'mobile_menu d_flex';
+  i += 1;
+  console.log(`has been clicked ${i} times`);
```

Comment:
Kindly do not commit `console.log` to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.